pkgs/by-name/ne/nerdfetch/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gitUpdater
+}:
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Test if it works.

Reply:
Yes I made the changes you requested and i can confirm that it works.